plugins/content/document/lib/operation_engines/KOperationEnginePdfCreator.php

```diff
@@ -96,19 +96,18 @@ public function operate(kOperator $operator = null, $inFilePath, $configFilePath
 		
 		$finalOutputPath = $this->outFilePath;
 		
-		if (($inputExtension == 'pdf') && ($this->data->flavorParamsOutput->readonly == true)){
-			$tmpFile = $this->outFilePath.'.pdf';
-		}else{
-			$tmpFile = kFile::replaceExt(basename($realInFilePath), 'pdf');
-			$tmpFile = dirname($this->outFilePath).'/'.$tmpFile;
-		}
+		$tmpFile = kFile::replaceExt(basename($realInFilePath), 'pdf');
```

Comment:
Read only functionality has been removed here - I see no reason for that, especially when it is not effecting the linux convertor.